### ISSUE TITLE
[a11y Support] <Team Name> - <Project><Detail>

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#92357]
> [!NOTE]
> **QUICK SUMMARY:** `PLACEHOLDER` abc

## Issue Description
`Placeholder` xyz

## About the Team

- **Team name**: 
- **Product name:** 
- **Product manager:**  
- **Slack Channel:**

### Supporting Artifacts 
- [Artifact 1](Placeholder1)
- [Artifact 2](Placeholder2)
- [Artifact 3](Placeholder3)

```[tasklist]
### a11y Tasks
- [ ] Collaborate `task`
- [ ] a11y Support `task`
- [ ] Document `task`
- [ ] Review `task`
```
```[tasklist]
### Acceptance Criteria
- [ ] Acceptance Criteria `Example 1`
- [ ] Acceptance Criteria `Example 2`
- [ ] Acceptance Criteria `Example 3`
```
```[tasklist]
## Ticket Updates (CAIA Internal)
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Naomi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
```

```[tasklist]
### Related Tickets
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/55956
- [ ] #ticketnumbergoeshere1
- [ ] #ticketnumbergoeshere2
- [ ] #ticketnumbergoeshere3
```

**Comment by sara-amanda** [comment]
## a11y Internal Ticket Template Work

For tickets we spin up ourselves, I have a `draft` here of a ticket template, we might consider. 

Feedback welcomed!

**Comment by rileyorr** [comment]
I love this format and I know I already messaged you this but wanted to publicly say I like the note section at the top to provide a quick overview before drilling into the specifics. 

I wonder what folks think about instead of the 'Product manager' label under 'About the team' it's like 'Team contact' or something? In my experience I have mostly been chatting with the designer directly so I would think to list them rather than the team's product manager. But I could see other scenarios where it makes sense to list the product manager!

**Comment by rileyorr** [comment]
@sara-amanda as part of the acceptance criteria will this ticket template be added as one of the template options when creating a new issue? Right now I've just been coming here and copying everything when I make a new ticket

**Comment by sara-amanda** [comment]
## Ticket Template Published

- **Location**: [Choose New Issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new/choose)
- **Title**: Internal CAIA a11y Short Ticket
- **Description**: Template for CAIA a11y members to utilize, when needing to create a ticket oustide the intake process. 
- **Link**: https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?template=CAIA-Internal-A11y-Short-Ticket.md

![Image](https://github.com/user-attachments/assets/006311af-82a3-4207-8ffb-8ad3f0c69062)
